config/samples/kafka_v1beta2_kafka.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kafka.jaegertracing.io/v1beta2
```

Comment:
this file should not be here and all references to `kafka.jaegertracing.io` should be removed